docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -106,6 +106,41 @@ Refer to
 [the advanced section of the installation guide](./10-installing-kargo.md#advanced-installation)
 for more details.
 
+While doing this you must ensure that the Kargo controller has the appropriate
+permissions to read `Secret` resources in these namespaces. This is necessary because,
+as of Kargo v1.0.0, the controller does not have cluster-wide access to `Secret`s. Permissions
+are granted dynamically on a per-Project basis as `Projects` are created.
+
+You can do so by creating your own `RoleBinding`s to permit the Kargo
+ controller(s) to read Secret(s) from each of those namespaces:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: kargo-controller-read-secrets
+  namespace: kargo
+subjects:
+- kind: ServiceAccount
+  name: kargo-controller
+  namespace: kargo
+roleRef:
+  kind: Role
+  name: kargo-controller-read-secrets
+  apiGroup: rbac.authorization.k8s.io
+```
+
+OR
+
+(Not recommended) Set `controller.serviceAccount.clusterWideSecretReadingEnabled` 
+setting to `true` in Kargo's Helm chart.
+
+:::warning
+It is important to understand the security implications of this feature. Any
+credentials stored in a global credentials `Namespace` will be available to
+_all_ Kargo projects.
+:::
```

Comment:
I know you moved this warning up from farther below, but this isn't the explanation for why `controller.serviceAccount.clusterWideSecretReadingEnabled=true` is highly discouraged. I would move this back to where it was...

Better yet, just remove it. _The whole section is about enabling global access to certain credentials._ It probably doesn't need repeating.

As for the warning text that _does_ belong here:

:::warning
Setting `controller.serviceAccount.clusterWideSecretReadingEnabled` setting to
`true` during Kargo installation will grant Kargo controllers cluster-wide read
permission on `Secret` resources.

__This is highly discouraged, especially in sharded environments where this
permission would have the undesirable effect of granting remote Kargo
controllers read permissions on all `Secret`s throughout the Kargo control
plane's cluster -- including `Secret`s having nothing to do with Kargo.__
:::